src/lib/components/workflow/workflows-summary-configurable-table/filterable-table-cell.svelte

```diff
@@ -27,6 +28,16 @@
     type = SEARCH_ATTRIBUTE_TYPE.KEYWORD,
   }: Props = $props();
 
+  const truncateRunId = (runId: string): string => {
+    if (runId.length > 11) {
+      return `${runId.slice(0, 4)}....${runId.slice(-4)}`;
```

Comment:
Let's use 3 ..., not 4

Reply:
Sounds good, I know that 3 is more standard and we use 3 in the namespace, but there were 4 in the designs so I wasn't sure if that was a big deal.